docs/API.md

```diff
@@ -499,3 +499,37 @@ tracer.use('express', {
 Right now this functionality is limited to Web frameworks.
 
 More information on which hooks are supported for each integration can be found in each individual [plugins](./modules/plugins.html).
+
+<h3 id="set-user">User Identification</h3>
+
+The tracer provides a convenience function to link an actor to a trace. For example to correlate users to web requests.
+You have to pass an object with at least an `id` property.
+
+For example:
+
+```javascript
+const tracer = require('dd-trace').init()
+
+function handle () {
+  tracer.setUser({
+    id: '123456789', // required
+     // all other fields are optional
+    email: 'jane.doe@example.com',
+    name: 'Jane Doe',
+    // abitrary fields are also accepted
+    custom_tag: 'custom data'
+  })
+}
+```
+
+List of standardized tags:
```

Comment:
If we don't recommend setting the tags directly, should this be here?

Reply:
Did you push?

Reply:
Yeah it's at the bottom of the file. Would you rather have it before the table ?
https://github.com/DataDog/dd-trace-js/pull/1902/files#diff-b57590968a12cee85a37c1b91d8cc7092cd8b68e50b242c39a178121db82a797R537

Reply:
What I meant is to remove the table completely.

Reply:
Oh ok, I wrote it as a reference to what fields you can pass to `setUser()`, but I guess it's redundant with the typescript definitions ?

Reply:
I can also just rename the table as "list of fields you can pass to setUser()" and remove the "tags" wording